diff -r b110654707d1 Makefile.am
--- a/Makefile.am	Mon Dec 09 21:34:15 2013 +0100
+++ b/Makefile.am	Fri Dec 20 02:51:36 2013 -0500
@@ -158,8 +158,8 @@
 # Native MacOS packaging
 # ----------------------
 install-exec-local:: 
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(pkgdatadir)/$(VERSION)
-	tar cvf - $(FF_EXAMPLES_FILES)| (cd $(DESTDIR)$(pkgdatadir)/$(VERSION); tar xvf -)
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(pkgdatadir)
+	tar cvf - $(FF_EXAMPLES_FILES)| (cd $(DESTDIR)$(pkgdatadir); tar xvf -)
 	$(mkinstalldirs) -m 755 $(DESTDIR)${bindir}
 	-test `uname` = Darwin && $(INSTALL_SCRIPT) FreeFem++-CoCoa  $(DESTDIR)${bindir}
 
diff -r b110654707d1 examples++-3d/Makefile.am
--- a/examples++-3d/Makefile.am	Mon Dec 09 21:34:15 2013 +0100
+++ b/examples++-3d/Makefile.am	Fri Dec 20 02:51:36 2013 -0500
@@ -35,8 +35,8 @@
 	echo loadpath += \"./\" >>freefem++.pref
 3d-Leman.edp:freefem++.pref
 install-exec-local:: 
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/idp
-	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(ff_prefix_dir)/idp
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(libdir)/freefem++/idp
+	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(libdir)/freefem++/idp
 
 
 
diff -r b110654707d1 examples++-load/Makefile.am
--- a/examples++-load/Makefile.am	Mon Dec 09 21:34:15 2013 +0100
+++ b/examples++-load/Makefile.am	Fri Dec 20 02:51:36 2013 -0500
@@ -195,24 +195,24 @@
 freefem++.pref:
 	echo loadpath = \"./\" >freefem++.pref
 install-exec-local:: load_compile WHERE_LIBRARY
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/lib
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/include
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/etc
-	$(INSTALL_DATA)  clapack.h ppmimg.h  bmo.hpp msh3.hpp pcm.hpp  include/*   $(DESTDIR)$(ff_prefix_dir)/include
-	$(INSTALL)  -m 555 $(LIST_COMPILE) $(DESTDIR)$(ff_prefix_dir)/lib
-	$(INSTALL)  -m 555 WHERE_LIBRARY-config WHERE_LIBRARY  $(DESTDIR)$(ff_prefix_dir)/lib
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(libdir)/freefem++
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(includedir)/freefem++
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(sysconfdir)
+	$(INSTALL_DATA)  clapack.h ppmimg.h  bmo.hpp msh3.hpp pcm.hpp  include/*   $(DESTDIR)$(includedir)/freefem++
+	$(INSTALL)  -m 555 $(LIST_COMPILE) $(DESTDIR)$(libdir)/freefem++
+	$(INSTALL)  -m 555 WHERE_LIBRARY-config WHERE_LIBRARY  $(DESTDIR)$(libdir)/freefem++
 	for i in  WHERE_LIBRARY-config WHERE_LIBRARY WHERE_LIBRARY-download; do \
-	  sed <$$i >$(DESTDIR)$(ff_prefix_dir)/lib/$$i 's#$(abs_top_builddir)/download#$(ff_prefix_dir)#' ;\
+	  sed <$$i >$(DESTDIR)$(libdir)/freefem++/$$i 's#$(abs_top_builddir)/download#$(prefix)#' ;\
 	 done
-	echo loadpath += \"./\"  >$(DESTDIR)$(ff_prefix_dir)/etc/freefem++.pref
-	echo loadpath += \"$(ff_prefix_dir)/lib\"  >>$(DESTDIR)$(ff_prefix_dir)/etc/freefem++.pref
-	echo includepath += \"$(ff_prefix_dir)/idp\"  >>$(DESTDIR)$(ff_prefix_dir)/etc/freefem++.pref
+	echo loadpath += \"./\"  >$(DESTDIR)$(sysconfdir)/freefem++.pref
+	echo loadpath += \"$(libdir)/freefem++\"  >>$(DESTDIR)$(sysconfdir)/freefem++.pref
+	echo includepath += \"$(libdir)/freefem++/idp\"  >>$(DESTDIR)$(sysconfdir)/freefem++.pref
 	$(INSTALL_SCRIPT) ff-c++  $(DESTDIR)${bindir}
 	$(INSTALL_SCRIPT) ff-pkg-download  $(DESTDIR)${bindir}
 	$(INSTALL_SCRIPT) ff-get-dep  $(DESTDIR)${bindir}
 ##	$(DESTDIR)${bindir}/ff-pkg-download $(DESTDIR) >$(DESTDIR)$(ff_prefix_dir)/lib/WHERE_LIBRARY-download
 	-for i in $(LIST_COMPILE_PKG); do \
-	 if [ -f $$i ] ; then 	$(INSTALL)  -m 555 $$i $(DESTDIR)$(ff_prefix_dir)/lib; fi; done
+	 if [ -f $$i ] ; then 	$(INSTALL)  -m 555 $$i $(DESTDIR)$(libdir)/freefem++; fi; done
 
 
 
diff -r b110654707d1 examples++-mpi/Makefile.am
--- a/examples++-mpi/Makefile.am	Mon Dec 09 21:34:15 2013 +0100
+++ b/examples++-mpi/Makefile.am	Fri Dec 20 02:51:36 2013 -0500
@@ -84,12 +84,12 @@
 	../examples++-load/ff-c++ -auto -mpi $<
 
 install-exec-local:: 
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/lib
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/include
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/etc
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(libdir)/freefem++
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(includedir)/freefem++
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(sysconfdir)
 	-for i in $(LIST_COMPILE); do \
-	 if [ -f $$i ] ; then 	$(INSTALL)  -m 555 $$i $(DESTDIR)$(ff_prefix_dir)/lib; fi; done
-	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(ff_prefix_dir)/idp
+	 if [ -f $$i ] ; then 	$(INSTALL)  -m 555 $$i $(DESTDIR)$(libdir)/freefem++; fi; done
+	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(libdir)/freefem++/idp
 
 
 freefem++.pref:Makefile
diff -r b110654707d1 examples++-tutorial/Makefile.am
--- a/examples++-tutorial/Makefile.am	Mon Dec 09 21:34:15 2013 +0100
+++ b/examples++-tutorial/Makefile.am	Fri Dec 20 02:51:36 2013 -0500
@@ -22,8 +22,8 @@
 Ref: makeref.edp freefem++.pref
 	../src/nw/FreeFem++-nw makeref.edp
 install-exec-local:: 
-	$(mkinstalldirs) -m 755 $(DESTDIR)$(ff_prefix_dir)/idp
-	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(ff_prefix_dir)/idp
+	$(mkinstalldirs) -m 755 $(DESTDIR)$(libdir)/freefem++/idp
+	$(INSTALL)  -m 555  $(LIST_IDP)  $(DESTDIR)$(libdir)/freefem++/idp
 
 
 makeref.edp: regtests.m4 ../regtests.m4
