diff -aur a/examples++-load/NewSolver.cpp b/examples++-load/NewSolver.cpp
--- a/examples++-load/NewSolver.cpp	2013-07-10 08:48:33.000000000 -0400
+++ b/examples++-load/NewSolver.cpp	2014-09-22 23:16:16.736455835 -0400
@@ -13,6 +13,7 @@
 #include "MatriceCreuse_tpl.hpp"
 
  
+#define HAVE_LIBUMFPACK
 #ifdef HAVE_LIBUMFPACK
 extern "C" {
 #ifdef HAVE_UMFPACK_H
diff -aur a/examples++-load/UMFPACK64.cpp b/examples++-load/UMFPACK64.cpp
--- a/examples++-load/UMFPACK64.cpp	2013-07-10 08:48:33.000000000 -0400
+++ b/examples++-load/UMFPACK64.cpp	2014-09-22 23:16:08.543986820 -0400
@@ -15,6 +15,7 @@
 
 #include <wchar.h>
  
+#define HAVE_LIBUMFPACK
 #ifdef HAVE_LIBUMFPACK
 extern "C" {
 #ifdef HAVE_UMFPACK_H
